### ISSUE TITLE
build fails with lens-4.15.2

### ISSUE THREAD

**Issue by juhp** [diagrams/active#20]
In the stackage nightly build:

```
Preprocessing library active-0.2.0.12...
[1 of 1] Compiling Data.Active      ( src/Data/Active.hs, dist/build/Data/Active.o )

src/Data/Active.hs:332:1: error:
    Duplicate instance declarations:
      instance Wrapped (MaybeApply f_auSr a_auSs)
        -- Defined at src/Data/Active.hs:332:1
      instance Wrapped (MaybeApply f a)
        -- Defined in ‘Control.Lens.Wrapped’

src/Data/Active.hs:332:1: error:
    Duplicate instance declarations:
      instance MaybeApply f_auSW a_auSX ~ t_auSY =>
               Rewrapped (MaybeApply f_auSZ a_auT0) t_auSY
        -- Defined at src/Data/Active.hs:332:1
      instance t ~ MaybeApply f' a' => Rewrapped (MaybeApply f a) t
        -- Defined in ‘Control.Lens.Wrapped’
```

**Comment by hvr** [comment]
I was about to this as well :-)

I've already revised the meta-data of affected releases on Hackage:

- https://hackage.haskell.org/package/active-0.2.0.12/revisions/

**Comment by byorgey** [comment]
Interesting, thanks.  I'll have to look into this.  The duplicate instances are being generated via TH (using `makeWrapped` from `lens`).

**Comment by byorgey** [comment]
This was fixed by @8cdf72f .
